setup.py

```diff
@@ -12,6 +12,10 @@
     url='https://github.com/imsweb/django-history-triggers',
     license='BSD',
     packages=find_packages(),
+    install_requires=[
+        django>=1.8,
```

Comment:
Django packages typically don't specify Django as a requirement. Anyone using it will pin their own version of Django anyway. Or if you do include it, it will need to be more specific than this, since there's presumably an upper limit as well.